_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
-msgstr ""
+msgstr "Beginne mit den einfachen Dingen: Vergewissere dich, dass dein Instrument/Mikrofon und deine Kopfhörer an die richtigen Buchsen angeschlossen sind. Vergewisseren dich, dass keine anderen Anwendungen wie Browser, Videokonferenz Programme usw. deine Soundkarte ebenfalls benutzen. Du solltest diese Anwendungen abschalten, wenn du Jamulus verwendest. Wenn das alles in Ordnung ist und das Problem weiterhin besteht, liegt wahrscheinlich ein Problem mit den Einstellungen deines Soundgeräts vor. Dies hängt von deiner speziellen Konfiguration ab (Plattform, Hardware, Software und Treiber), daher fragen am besten in [den Foren] (https://github.com/jamulussoftware/jamulus/discussions) nach."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:25
 #, no-wrap
 msgid "**Windows users (ASIO4ALL)**: If you’re using the ASIO4ALL driver have a look at the [ASIO4LL setup section](Installation-for-Windows#setting-up-asio4all)\n"
-msgstr ""
+msgstr "**Windows-Benutzer (ASIO4ALL)**: Wenn du den ASIO4ALL-Treiber verwendest, wirf einen Blick auf den [ASIO4LL setup section](Installation-for-Windows#ASIO4ALL einrichten)\n"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:27
 msgid "### Not seeing the headphone or microphone you just connected to your PC?"
-msgstr ""
+msgstr "### Du kannst den Kopfhörer oder das Mikrofon nicht sehen, den/das du gerade an deinen PC angeschlossen hast?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:29
 msgid "Just restart Jamulus with your device plugged in. Jamulus currently doesn't show devices which were plugged in after the sound system was loaded."
-msgstr ""
+msgstr "Starte Jamulus einfach neu, wenn deine Geräte eingesteckt sind. Jamulus zeigt derzeit keine Geräte an, die nach dem Laden des Soundsystems eingesteckt wurden."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:31
 msgid "### Your sound is stuttering"
-msgstr ""
+msgstr "### Dein Sound stottert"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:33
 msgid "Your audio device may not work with the buffer size you selected. Select a bigger buffer size in Jamulus's settings. If possible though, use another device, since large buffer sizes mean higher latency."
-msgstr ""
+msgstr "Dein Audiogerät funktioniert möglicherweise nicht mit der von dir gewählten Puffergröße. Wähle in den Einstellungen von Jamulus eine größere Puffergröße. Wenn möglich, solltest du ein anderes Gerät verwenden, da große Puffergrößen eine höhere Latenz bedeuten."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:35
 msgid "### You all sound OK, but it's difficult to keep together"
-msgstr ""
+msgstr "### Ihr klingt alle ganz gut, aber es ist schwierig, zusammenzubleiben"
```

Comment:
Not quite sure about that unfortunately. Ihr sounds a bit strange although it's perfect correct here.

Reply:
Ihr klingt alle ganz gut, aber es ist schwierig, zusammenzubleiben"

Maybe better:

Ihr klingt alle ganz gut, aber es ist schwierig, **im Takt zu bleiben**"

Reply:
Yeah. Maybe better.

Reply:
I will collect all changes & comments and then create new pull request, ok?

Reply:
No, please don't make a new PR. You can just add commits to this branch and they'll be added to this PR.

I tested the website locally yesterday and found the 'de' version won't load, but after some head scratching found the problem: I'm on a mobile phone now so can't really point it out properly, but will do so tonight.

Btw yes, thanks for the big effort!

Reply:
Ok
Will change file and then commit only

Reply:
Please tell me how to view the new DE Web Pages.
Would like to check if all links are working...

Reply:
Check this out: https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll

You still won't be able to see the German translation though, there are a couple of small corrections that you need to make.

Reply:
Ok, what kind of changes?

Reply:
I think you can also download the Website.zip file from the Checks tab on this Pull request and serve it via any local web server.

Reply:
Downloaded the ZIP file
But ZIP file contains only "symlinks" for index-de,...

Reply:
If you've installed jekyll following the instructions in the above link you have to navigate in your terminal to the _site folder that results from unzipping Website.zip and run `bundle exec jekyll serve`. Then, in your browser, enter `http://localhost:4000` in the address field. The Jamulus website should now be displayed.

Reply:
There is a warning

`/Library/Ruby/Gems/2.6.0/gems/jekyll-4.2.1/lib/jekyll/utils.rb:141:in `initialize': No such file or directory @ rb_sysopen - /Users/reiny/Documents/GitHub/jamuluswebsite-test-de/1-de-index.html (/Library/Ruby/Gems/2.6.0/gems/jekyll-4.2.1/lib/jekyll/utils.rb:141:in `initialize'/Library/Ruby/Gems/2.6.0/gems/jekyll-4.2.1/lib/jekyll/utils.rb:141:in `initialize': : No such file or directory @ rb_sysopen - /Users/reiny/Documents/GitHub/jamuluswebsite-test-de/1-de-index.htmlNo such file or directory @ rb_sysopen - /Users/reiny/Documents/GitHub/jamuluswebsite-test-de/1-de-index.html ( (Errno::ENOENTErrno::ENOENT))`

final message
` Auto-regeneration: enabled for '/Users/reiny/Documents/GitHub/jamuluswebsite-test-de'
        ** ERROR: directory is already being watched! **

        Directory: /Users/reiny/Documents/GitHub/jamuluswebsite-test-de/assets/img

        is already being watched through: /Users/reiny/Documents/GitHub/jamuluswebsite-test-de/assets/img

        MORE INFO: https://github.com/guard/listen/blob/master/README.md
    Server address: http://127.0.0.1:4000
  Server running... press ctrl-c to stop.`

Reply:
EN Page is loading locally ok
DE Page not

`[2022-05-16 10:50:45] ERROR `/de/' not found.
[2022-05-16 10:50:57] ERROR `/de/wiki/Getting-Started' not found.
[2022-05-16 10:51:03] ERROR `/de/wiki/Installation-for-Linux' not found.
[2022-05-16 10:51:16] ERROR `/de/wiki/Installation-for-Linux' not found.
[2022-05-16 10:51:28] ERROR `/de/wiki/Getting-Started' not found.`

Reply:
Yes, you first need to correct and commit the fixed quotes. A new Website.zip file will be created.

Reply:
Maybe you have to trigger a re-fresh

<img width="507" alt="grafik" src="https://user-images.githubusercontent.com/77500932/168558214-48374a48-6066-4964-be42-54acd240b9b4.png">

Reply:
I am using GitHub Desktop
Submitted there.
Now its clear I also have to trigger some commands in terminal to actually trigger the re-creation

Reply:
Just did it now, should be ready in a couple of minutes.

Reply:
Thanks!

Compared  DE and NL 

Found in NL: #**|** "<figure>\n"

<img width="249" alt="grafik" src="https://user-images.githubusercontent.com/77500932/168649918-7c781a5a-ec7b-4e0e-87b7-438da3149b87.png">